tools/system_libs.py

```diff
@@ -547,59 +547,64 @@ def get_default_variation(cls, **kwargs):
     return super(MTLibrary, cls).get_default_variation(is_mt=shared.Settings.USE_PTHREADS, **kwargs)
 
 
+class exceptions(object):
+  none = 0 # Don't use exception support
+  emscripten = 1
+  wasm = 2
```

Comment:
I think it might be worth adding some docs on this class. Maybe a general overview of the 3 modes and what they mean, and the long-term plans. That information doesn't seem to be anywhere else.

Reply:
Done.